src/features/silent_walk.cpp

```diff
@@ -0,0 +1,42 @@
+#include "features.h"
+#include "../settings/globals.h"
+#include "../helpers/console.h"
+#include "../helpers/input.h"
+
+namespace silent_walk
+{
+	bool is_enabled(CUserCmd* cmd)
```

Comment:
Why is CUserCmd* parameter inside the is_enabled function definition when its not used???